src/utils/user-list-helpers.js

```diff
@@ -0,0 +1,190 @@
+import {
+  getSolidDataset,
+  createThing,
+  buildThing,
+  setThing,
+  createSolidDataset,
+  getThingAll,
+  saveSolidDatasetAt,
+  getStringNoLocale,
+  getUrl,
+  removeThing,
+  getPodUrlAll
+} from '@inrupt/solid-client';
+
+import { createDocAclForUser } from './session-helper';
+
+import { RDF_PREDICATES } from '../constants';
+
+/**
+ * @typedef {import('@inrupt/solid-ui-react').SessionContext} Session
+ */
+
+/**
+ * @typedef {import("../typedefs").userListObject} userListObject
+ */
+
+const createUsersList = async (session, usersListUrl) => {
+  const newUsersList = buildThing(createThing({ name: 'userlist' }))
+    .addStringNoLocale(RDF_PREDICATES.name, 'Users List')
+    .addStringNoLocale(RDF_PREDICATES.description, 'A list of users')
```

Comment:
There was originally a URL parameter pointing back to itself here. Deleted because it served no function.